inst/tests/tests.Rraw

```diff
@@ -18702,3 +18704,14 @@ test(2234.7, DT[, min(.SD), by=c(.I)], data.table(I=1L:5L, V1=c(1L, 2L, 3L, 2L,
 test(2234.8, DT[, min(.SD), by=.I%%2L], error="by.*contains .I.*supported")  # would be nice to support in future; i.e. by odd/even rows, and by=(.I+1L)%/%2L for pairs of rows; i.e. any expression of .I
 test(2234.9, DT[, min(.SD), by=somefun(.I)], error="by.*contains .I.*supported")
 
+# move IDate calculating from POSIXlt to C, add yearquarter #649
+x = c("1111-11-11", "2019-01-01", "2019-02-28", "2019-03-01", "2019-12-31", "2020-02-29", "2020-03-01", "2020-12-31")
```

Comment:
test a date far in the future too?

Reply:
Shouldn't apply here since we always have days since 1970-01-01 or 2000-03-01 (in the calculation itself) but will add a testcase.

Reply:
agreed, mainly had it in mind as a regression test